providers/tests/integration/trino/hooks/test_trino.py

```diff
@@ -50,7 +50,7 @@ def test_should_record_records_with_kerberos_auth(self):
 
     @mock.patch.dict("os.environ", AIRFLOW_CONN_TRINO_DEFAULT="trino://airflow@trino:8080/")
     def test_openlineage_methods(self):
-        op = TrinoOperator(task_id="trino_test", sql="SELECT name FROM tpch.sf1.customer LIMIT 3")
+        op = SQLExecuteQueryOperator(task_id="trino_test", sql="SELECT name FROM tpch.sf1.customer LIMIT 3")
```

Comment:
we should add `conn_id="trino_default"` as this test is failing now.

Reply:
Ya I'll correct that id